geometry/spline.cpp

```diff
@@ -52,20 +60,20 @@ void Spline::ReplacePoint(PointD const & pt)
   AddPoint(pt);
 }
 
-bool Spline::IsPrelonging(PointD const & pt)
+bool Spline::IsProlonging(PointD const & pt) const
```

Comment:
its used in apply_feature_functors only and contains similar min length condition which is redundant;

I think makes sense to do both length and angle comparisons in one place only - either apply_feature_functors or here